apps/web/lang/en.json

```diff
@@ -488,5 +488,17 @@
     "contactSend": "Send request",
     "clearAll": "Clear all",
     "privacyPolicy": "I hereby confirm that I have read the {privacyPolicy}."
+  },
+  "coupon": {
+    "name": "Coupon",
+    "title": "Have a coupon code?",
+    "enterCode": "Enter code",
+    "apply": "Apply",
+    "voucherApplied": "The coupon has been added successfully",
+    "voucherRemoved": "The coupon has been removed successfully.",
+    "voucherInvalid": "The coupon has already been used or is invalid.",
+    "error": {
+      "1070": "Coupon code is not valid."
```

Comment:
I would create a map for these codes as @maxiroellplenty suggested, if we place them in the language files, when an external developer wants to change a translation he would have to know which code to look for, which is not good. If we have a map with for example:

`"1070": "invalidCouponError"`

and then in this translation file

`"invalidCouponError": "Coupon code is not valid."`

I think it would provide more clarity for our external devs.